source/edge-server.txt

```diff
@@ -0,0 +1,126 @@
+   :alt: Atlas Device Sync Illustration
+
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: MongoDB + Sync - On Premises
+      :icon: /images/icons/General_FEATURES_IntelligentEdge3x.png
+      :icon-alt: An icon depicting a mobile phone connecting with the cloud
+
+      Edge server is a local instance of MongoDB paired with a sync server 
+      that runs on-premises. It brings real-time sync, disconnection 
+      tolerance, and conflict resolution to the location's devices.
+
+   .. card::
+      :headline: Optimize Bandwidth
+      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
+      :icon-alt: An icon depicting conflict resolution
+
+      Delta sync ensures that only fields with changes in a document are 
+      sent to Atlas. Mission-critical data is transferred efficiently while 
+      minimizing network congestion.
+
+   .. card::
+      :headline: Client Independence
+      :icon: /images/icons/Technical_REALM_Offline3x.png
+      :icon-alt: An icon depicting network loss
+
+      Edge clients can sync and access essential data independently, even 
+      when disconnected from the central database. Applications and devices 
+      in a connected infrastructure continue to function seamlessly, 
+      even with intermittent connectivity.
+
+.. kicker:: How it Works
+
+Edge Server Setup
+-----------------
+
+.. procedure::
+
+   .. step:: Create a Device Sync-Enabled App
+
+      Create an :ref:`App Services App <create-app>` with 
+      :ref:`Device Sync enabled <enable-sync>`.
+
+      To get started quickly, create a template app.
+
+      Once your App is created, coordinate with your Product or Account 
+      Representative to enable Edge Server.
+
+      .. button:: Start Building with a Template App
+         :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+
+   .. step:: Configure the Edge Server Host
+
+      Configure hardware on-site or a cloud server to host your Edge Server.
+      Refer to the documentation for required network access, hardware 
+      requirements, and supported operating systems.
+
+   .. step:: Install and Configure the Edge Server
+
+      Install the Edge Server on your hardware, complete a basic 
+      ``config.json`` with your configuration details, and install required
+      dependencies. Then, start the Edge Server.
+
+   .. step:: Configure an Authentication Provider
+
+      Enable an authentication provider to secure client connections to
+      the Edge Server.
+
+      Edge Server supports these authentication providers:
+
+      - :ref:`Anonymous Authentication <anonymous-authentication>`
+      - :ref:`Email/Password Authentication <email-password-authentication>`
+      - :ref:`Custom JWT Authentication <custom-jwt-authentication>`
+      - :ref:`API Key Authentication <api-key-authentication>`
+
+   .. step:: Connect to the Edge Server from a Client
```

Comment:
it would be worthwhile saying explicitly how (set the base_url field in your realm config, maybe link to some docs for this)

Reply:
Yeah, I'll add a link to the "Connect to Edge Server" page once the content there is built out.